docs/src/gettingstarted/introduction/own-code/cli-requirements.md

```diff
@@ -10,10 +10,6 @@ Now that you have created your free trial account, you are able to push your app
 
 Before you install it, there are some requirements that must be met first.
 
-## Git
+{{% cli-installation-requirements %}}
 
-[Git](/development/tools.md#git) is the open source version control system used by Platform.sh. Any change you make to your Platform.sh project will need to be committed via Git. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console](/administration/web/_index.md) for each project you create.
-
-Before getting started, make sure you [have Git installed](https://git-scm.com/) on your computer.
-
-{{< guide-buttons next="I have Git ready" >}}
+{{< guide-buttons next="I have met all the requirements" >}}
```

Comment:
:lady_beetle: 
I feel like a lot of the context is lost this way. There's no longer a generic introduction to what Git is and why it's useful.

That's why I wasn't sure that the same shortcode should be used in the generic installation and the getting started guide.

Reply:
The purpose of a tutorial in this context is to be self-contained. It's different than material elsewhere, where you want to worry about being repetitive and such. In this context, you want the tutorial to include everything necessary, which here means an explanation of what Git is.

The how-to guide to the CLI should assume basic knowledge, such as what Git is and why you'd need it installed. Really, it should assume you have Git installed, as otherwise you can't really work with Platform.sh, but it doesn't hurt to mention in passing. But the extra context doesn't work there, I'd say.